ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -176,7 +178,14 @@ export default class UnlockPage extends Component {
               </div>
             ) : null}
           </div>
-          <h1 className="unlock-page__title">{t('welcomeBack')}</h1>
+          <Text
+            as="h1"
+            variant={TextVariant.headingMd}
```

Comment:
I believe that the mapping in the issue is h1 -> displayMd not headingMd and perhaps that's why in your screenshot there is a large difference in the size of the fonts

Reply:
Okay will do it in a while.
Thanks

Reply:
resolved that. Can you check once?